### PR TITLE
feat: Backlog/align utils

### PR DESCRIPTION
**Pull request by henriknorin-ftrack**
Resolves : 

* CLICKUP-865bugxcq

- [ ] I have added automatic tests where applicable.
- [ ] The PR contains a description of what has been changed.
- [ ] The description contains manual test instructions.
- [ ] The PR contains update to the release notes.
- [X] The PR contains update to the documentation.

This PR has been tested on :

- [ ] Windows.
- [X] MacOs, Maya 2022 & 2023
- [ ] Linux.

## Changes

Split utils across DCCs to align with Unreal

## Test

Do a full cycle test in Maya

### CODE REVIEW

File: resource/bootstrap/userSetup.py [hunk 1]
```diff
@@ -4,7 +4,6 @@
 import logging
 import functools
 
-import maya.utils
```
Comment (lluisCM) [inline resource/bootstrap/userSetup.py:7]: Can't remove this as it's used

File: source/ftrack_connect_pipeline_maya/client/asset_manager.py [hunk 2]
```diff
@@ -9,7 +9,6 @@
 import ftrack_connect_pipeline_maya.constants as maya_constants
 
 from maya.app.general.mayaMixin import MayaQWidgetDockableMixin
-from ftrack_connect_pipeline_maya.utils.custom_commands import get_main_window
```
Comment (lluisCM) [inline source/ftrack_connect_pipeline_maya/client/asset_manager.py:12]: Is this not used anywhere?

File: source/ftrack_connect_pipeline_maya/client/change_context.py [hunk 3]
```diff
@@ -2,7 +2,6 @@
 # :copyright: Copyright (c) 2014-2022 ftrack
 
 from ftrack_connect_pipeline_qt.client import change_context
-from ftrack_connect_pipeline_maya.utils.custom_commands import init_maya
```
Comment (lluisCM) [inline source/ftrack_connect_pipeline_maya/client/change_context.py:5]: You are missing the import

**Review by lluisCM — CHANGES_REQUESTED** [review]
Please test it first and I'll review later.

File: source/ftrack_connect_pipeline_maya/client/asset_manager.py (hunk 2 above)
Comment (henriknorin-ftrack) [reply source/ftrack_connect_pipeline_maya/client/asset_manager.py:12]: No import is not used anymore, was there during a phase when I were debugging window modality in Maya

File: source/ftrack_connect_pipeline_maya/client/change_context.py (hunk 3 above)
Comment (henriknorin-ftrack) [reply source/ftrack_connect_pipeline_maya/client/change_context.py:5]: Well spotted, fixing

File: resource/bootstrap/userSetup.py (hunk 1 above)
Comment (henriknorin-ftrack) [reply resource/bootstrap/userSetup.py:7]: Import is not used anywere in code, only explicit through maya.utils.MayaGuiLogHandler but that does not need the import right?

**Review by lluisCM — APPROVED** [review]
LGTM

**Pull request merged**